src/components/Select.vue

```diff
@@ -318,7 +318,7 @@
           :multiple="multiple"
           :disabled="disabled"
         >
-          <span class="selected-tag" v-bind:key="option.index" :class="{ 'multiple':multiple }">
```

Comment:
I discovered that the v-select already has a CSS class which differentiates both.
To style the multiple input, I now use:
```
.v-select:not(.single) .selected-tag {
}
```